.vscode/settings.json

```diff
@@ -4,7 +4,7 @@
     "editor.formatOnSave": true,
     "editor.tabSize": 4,
     "editor.codeActionsOnSave": {
-      "source.organizeImports": true,
+      "source.organizeImports": "explicit"
```

Comment:
vscode keeps saving the file like this even if I revert it. So I assume it's vscode deprecating or changing a naming convention

Reply:
+1
I can confirm that my vscode updated the same change!